core/src/main/scala/akka/persistence/jdbc/query/javadsl/JdbcReadJournal.scala

```diff
@@ -23,6 +24,9 @@ class JdbcReadJournal(journal: ScalaJdbcReadJournal)
     with EventsByPersistenceIdQuery
     with CurrentEventsByTagQuery
     with EventsByTagQuery {
+
+  PluginVersionChecker.check()
```

Comment:
Also checking when loading the read-side component.

It's very common to start queries on application bootstrap. That will force the check in a very early stage.

Reply:
Actually isn't the java dsl using forwarding the calls to the scala dsl? I think checking only for the scala dsl should be sufficient

Reply:
Ok, thanks for the heads-up. I will check and adapt if necessary.